globals/macros.yaml

```diff
@@ -0,0 +1,230 @@
+          export ANSIBLE_CONFIG=/home/jenkins/quickstart/tripleo-quickstart/ansible.cfg
+          export SSH_CONFIG="/home/jenkins/quickstart/ssh.config.ansible"
+          export ANSIBLE_SSH_ARGS="-F ${SSH_CONFIG}"
+
+          . /home/jenkins/quickstart/bin/activate
+          ansible-playbook -v \
+          -e @./config/logs/oooq.yml \
+          -e artcl_collect_dir=${WORKSPACE}/logs \
+          ./tripleo-quickstart-nfv-collect-logs/playbooks/collect-logs.yml
+          deactivate
+
+- builder:
+    name: collect-logs-validate
+    builders:
+      - shell: |
+          export ANSIBLE_INVENTORY=/home/jenkins/quickstart/hosts
+          export ANSIBLE_CONFIG=/home/jenkins/quickstart/tripleo-quickstart/ansible.cfg
+          export SSH_CONFIG="/home/jenkins/quickstart/ssh.config.ansible"
+          export ANSIBLE_SSH_ARGS="-F ${SSH_CONFIG}"
+
+          . /home/jenkins/quickstart/bin/activate
+          ansible-playbook -v \
+          -e @./config/logs/tempest.yml \
+          -e artcl_collect_dir=${WORKSPACE}/logs \
+          --limit undercloud \
+          ./tripleo-quickstart-nfv-collect-logs/playbooks/collect-logs.yml
+          deactivate
+
+- builder:
+    name: collect-ara-results
+    builders:
+      - shell: |
+          . /home/jenkins/quickstart/bin/activate
+          export ARA_DATABASE="sqlite:///${WORKSPACE}/ara_database.sqlite"
+
+          # collect ara results
+          UNREACHABLE_RESULT=$(ara stats list -f value -c Unreachable)
+          FAILED_RESULT=$(ara stats list -f value -c Failed)
+          ara generate ${WORKSPACE}/ara_results || true
+          deactivate
+
+          # parse to get final job result
+          for result in $UNREACHABLE_RESULT ; do   if [ $result != '0' ]; then     exit 1;   fi; done
+          for result in $FAILED_RESULT ; do   if [ $result != '0' ]; then     exit 1;   fi; done
+
+- builder:
+    name: install-browbeat
+    builders:
+      - shell: |
+            . /home/jenkins/quickstart/bin/activate
+            export ANSIBLE_INVENTORY=/home/jenkins/quickstart/hosts
+            export ANSIBLE_CONFIG=/home/jenkins/quickstart/triple-quickstart/ansible.cfg
+            export SSH_CONFIG=/home/jenkins/quickstart/ssh.config.ansible
+            export 'ANSIBLE_SSH_ARGS=-F /home/jenkins/quickstart/ssh.config.ansible'
+
+            ansible-playbook -v \
+            -e @./config/tests/browbeat.yml \
+            ./browbeat/ansible/install/browbeat.yml
+
+            deactivate
+
+- builder:
+    name: download-oooq-images
+    builders:
+      - shell: |
+          IMAGE_HASH=$(curl -s https://images.rdoproject.org/{release_version}/delorean/consistent/stable/delorean_hash.txt)
+          IMAGE_ROOT=/var/lib/jenkins/images/{release_version}
+          IMAGE_DIR=$IMAGE_ROOT/$IMAGE_HASH
+          mkdir -p $IMAGE_DIR
+          pushd $IMAGE_ROOT
+          pushd $IMAGE_DIR
+          wget https://images.rdoproject.org/{release_version}/delorean/consistent/stable/ironic-python-agent.tar
```

Comment:
It is not mandatory, but I suppose we could use 'curl' instead of 'wget' because tripleo-quickstart uses curl. 
If we just use curl only, user does not need to install wget additionally.